examples/platforms/k32w/jn5189/Makefile.am

```diff
@@ -0,0 +1,141 @@
+#
+#  Copyright (c) 2019, The OpenThread Authors.
+#  Copyright (c) 2019, NXP.
```

Comment:
Can this simply be "The OpenThread Authors."?
```suggestion
#  Copyright (c) 2019, The OpenThread Authors.
```

Reply:
Solved.